### DISCUSSION TITLE
Syntax for multi-line figure captions in Python code cells

### DISCUSSION THREAD

**Discussion by astoeriko** [quarto-dev/quarto-cli#5407] [Q&A] [answered]
Some of my figure captions are quite long, so I would like to split them into several lines in the source code for better readability. What is the right syntax for this? Based on the instructions for mermaid diagrams that I found [here](https://github.com/quarto-dev/quarto-cli/discussions/1816) I tried as follows:
        
    ```{python}
    #| fig-cap: |
    #|   This is a caption for my figure.
    #|   It is long, so I put it onto two lines
    import numpy as np
    import matplotlib.pyplot as plt
    plt.plot(np.arange(10))
    ```
But when I run `quarto render`, I get an error:

```
Executing 'test.ipynb'
  Cell 1/1...ERROR: 

None of the known patterns match for   It is long, so I put it onto two lines
```
It runs fine if I put all the caption onto one line, but this is impractical with caption that are several sentences long.

    ```{python}
    #| fig-cap: "This is a caption for my figure. It is long, so I put it onto two lines."
    import numpy as np
    import matplotlib.pyplot as plt
    plt.plot(np.arange(10))
    ```

**Comment by cderv** [comment, marked as answer]
It seems like an issue with our parser so I'll move this to an issue #5413
